index.html

```diff
@@ -4,12 +4,12 @@
         <meta charset="utf-8">
         <meta name="viewport" content="initial-scale=1,width=device-width">
         <title>Wo ist Markt?</title>
-        <link rel="stylesheet" href="lib/normalize/normalize.css"/>
-        <link rel="stylesheet" href="lib/leaflet/leaflet.css"/>
-        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css"/>
-        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css"/>
-        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
-        <link href="lib/select2/select2.min.css" rel="stylesheet" />
+        <link rel="stylesheet" href="lib/normalize/normalize.css">
+        <link rel="stylesheet" href="lib/leaflet/leaflet.css">
+        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css">
+        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+        <link href="lib/select2/select2.min.css" rel="stylesheet">
```

Comment:
@dirkschumacher What did you use to lint the HTML? I tried [html-tidy5](https://github.com/htacg/tidy-html5) with [SublimeLinter-html-tidy](https://github.com/SublimeLinter/SublimeLinter-html-tidy). I did not get any warning nor error though.